packages/dso-toolkit/src/components/plekinfo-card/plekinfo-card.models.ts

```diff
@@ -1,14 +1,19 @@
 import { Renvooi } from "../renvooi/renvooi.models.js";
 import { Label } from "../label/label.models.js";
+import { Button } from "../button";
+import { Toggletip } from "../toggletip/toggletip.models";
+import { SlideToggle } from "../slide-toggle";
 
 export interface PlekinfoCard<TemplateFnReturnType> {
   label: Renvooi | string;
   href: string;
+  targetBlank: boolean;
   active?: boolean;
   meta?: Label;
   content?: TemplateFnReturnType;
   symbool?: TemplateFnReturnType;
   wijzigactie?: PlekinfoWijzigactie;
+  interactions?: Array<Button | Label | Toggletip<TemplateFnReturnType> | SlideToggle>;
```

Comment:
Geen noodzaak om deze een Array te laten zijn. Er is slecht een `SlideToggle` of niets.